frontend/src/lib/components/InstanceSettings.svelte

```diff
@@ -269,6 +277,25 @@
 		}
 		return true
 	}
+
+	let snowflakeAccountIdentifier = ''
+
+	$: snowflakeAccountIdentifier && setupSnowflakeUrls()
+
+	function setupSnowflakeUrls() {
+		// strip all whitespaces from account identifier
+		snowflakeAccountIdentifier = snowflakeAccountIdentifier.replace(/\s/g, '')
```

Comment:
this is creating a loop but overall the whole reactivity doesn't make much sense, it seems snowflakeAccountIdentifiers can only be modified in one place, you could just call setupSnowflaeUrls from there directly

Reply:
actually it can be set in 2 places, but I would encapsulate this in a separate Snowflake component, avoid the loop (the loop can be avoided by doing a check if there is a diff before doing the assignment)